test/shopify-cli/git_test.rb

```diff
@@ -176,7 +176,7 @@ def test_sparse_checkout
 
     def in_repo
       Dir.mktmpdir do |dir|
-        system("git init #{Shellwords.escape(dir)}> /dev/null")
+        system("git init --initial-branch=main #{Shellwords.escape(dir)}> /dev/null")
```

Comment:
I'm adding this one to eliminate some warnings that surface when running the tests.